### ISSUE TITLE
Mobile app signup without grafana cloud doesn't work.

### ISSUE THREAD

**Issue by justinhauer** [grafana/oncall#4561]
### What went wrong?

Ran developer stack using docker compose default settings. Downloaded iOS app. try to connect mobile app to grafana running locally on network.

- prefilled url defaults to https, this should be redacted from the field, when redirecting to grafana, the token previously created to sign in doesn't work
- the QR code option completely absent from the grafana UI
- syncing app to hosted grafana instance seems like an afterthought compared to cloud and seemingly doesn't work on a local network? 
- instructions have become outdated in UI and redirects user to user settings instead of on call section.

### How do we reproduce it?

Look in section above ^. I ask that someone from the onCall team run it hosted and attempt to sync the mobile app without a grafana cloud account. In environments where calls cannot be made to cloud for security reasons, this seems to be challenging if not possible in current build...

### Grafana OnCall Version

v1.7.1

### Product Area

Mobile App

### Grafana OnCall Platform?

None

### User's Browser?

Safari, mobile iOS 17 and macOS safari version 17.5

**Comment by justinhauer** [comment]
Hi @joeyorlando, is this an instance of the documentation getting outdated with the latest release, or is this something where there's active bugs, or is it impossible to use onCall and sync to a mobile device (for notifications just to the app) without the onCall cloud account?

Thanks!

Justin

**Comment by karmops** [comment]
anybody?

**Comment by karmops** [comment]
![image](https://github.com/user-attachments/assets/37362aea-1d4b-4a0a-a71f-a4f1995e10e6)

true or false?

**Comment by imtoori** [comment]
Hey you can follow these instructions to setup the app in open source https://grafana.com/docs/oncall/latest/set-up/open-source/?pg=oss-oncall&plcmt=hero-btn-3

I think in a case like this one 
> In environments where calls cannot be made to cloud for security reasons

I think there is nothing we can do, but I'll check with the backend team

**Comment by karmops** [comment]
![image](https://github.com/user-attachments/assets/eaca6d6e-9676-4dcb-a764-ef6a139beed4)


thank you sir, gonna try it.

**Comment by imtoori** [comment]
In general, a Grafana Cloud account is required. 
We need to send push notifications from our backends.

**Comment by karmops** [comment]
yeah, it is not possible to live OnCall OSS without Grafana Cloud. Even you deploying all the dependecies (database, rabbitmq, redis, oncall) you still need the Grafana Cloud to use Grafana OnCall Mobile App (sadly).

**Comment by imtoori** [comment]
Yes, unfortunately there is no way to send push notifications without going via the oncall backend
